libs/api-bundle/src/Attribute/ManageApiTokenAuth.php

```diff
@@ -11,6 +11,7 @@ class ManageApiTokenAuth implements AuthAttributeInterface
 {
     public function __construct(
         readonly array $scopes = [],
+        readonly public bool $isSuperAdmin = false,
```

Comment:
Kdyz bych ten atribut pouzil `#[ManageApiToken(isSuperAdmin: false)]`, asi bych to cetl jako ze ten user nesmi byt super-admin. Nechces to udelat jako tri-state option?
* `null` (default) - neres jestli je super admin nebo ne
* `true` - musi byt super-admin
* `false` - nesmi byt super-admin

Reply:
Taky jsem nad tím přemýšlel, ještě mně napadla varianta nechat to a přejmenovat na něco jako `hasToBeSuperAdmin`... Záleží jestli je existuje potřeba "nesmi byt super-admin"

Reply:
ok

Reply:
@pepamartinec wtf? https://dev.azure.com/keboola-dev/Platform%20Libraries/_build/results?buildId=44002&view=logs&j=f6c039e1-eff6-59f3-d2f1-4d1cdea0e360&t=787e3f66-7f51-5366-f655-ce1ea7399fc7

Reply:
Dobrý, už to mám